client/src/components/Cohort.jsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Repo from './Repo';
+import CheckboxList from './CheckboxList';
+
+class Cohort extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Probably worth pulling repos in from an external module as well in a future iteration.